metatensor-torch/src/register.cpp

```diff
@@ -382,6 +382,7 @@ TORCH_LIBRARY(metatensor, m) {
         )
         .def("__repr__", &ModelMetadataHolder::print)
         .def("__str__", &ModelMetadataHolder::print)
+        .def("print", &ModelMetadataHolder::print)
```

Comment:
this is registered as `__str__`; so you can do `print(metadata)` from Python and you don't need to do `print(metadata.print())`

Reply:
I know, but then the docs are wrong (?)

Reply:
I just wanted to print, looked up the docs, did `print()`, didn't work. Should we change the Python docs?

Reply:
yes!